### PR TITLE
W-14153243-PicklistMappingUpdate-ESB

### PR DESCRIPTION
**Pull request by emileeburkeVT**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: en/mulesoft/modules/ROOT/pages/ms_composer_flows.adoc [hunk 1]
```diff
@@ -480,7 +480,7 @@ When mapping source data fields:
 To map source data fields to target fields:
 
 . From the *Composer Flows* tab, open a flow and click *Edit*.
-. In an action card, in a field that supports *Pick from List*, select *Picklist Mapping* from the menu. +
+. In an action card, in a field that supports *Pick from List* or any `String` field, select *Picklist Mapping* from the menu. +
```
Comment (hannanelson) [inline en/mulesoft/modules/ROOT/pages/ms_composer_flows.adoc:483]: Is String a data type here or a field? If the former, I think our style is no font change. If the latter, then bold.
https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference#MuleSoftCXWritingStyleReference-fontchoices

File: en/mulesoft/modules/ROOT/pages/ms_composer_flows.adoc [hunk 2]
```diff
@@ -494,7 +494,10 @@ To create a new picklist:
 . Optionally, in the *Description* field, enter a description for the picklist. The following description is populated automatically based on your source and target systems: +
 `This maps [Source Field Name] from [Source Connection Name] to [Target Field Name] from [Target Connection Name]`
 . In the table, select the source fields and their target mappings.
-. Optionally, to add additional field mappings, click *Add Row* and enter the source field values manually (with a row representing each value) and select the target mappings from the dropdown.
+. Optionally, to add additional field mappings, click *Add Row* and if the additional field mapping is a:
+.. *Pick from List* field, enter the source field values manually (with a row representing each value) and select the target mappings from the dropdown.
+.. String field, manually define the value in the target field.
```
Comment (hannanelson) [inline en/mulesoft/modules/ROOT/pages/ms_composer_flows.adoc:499]: This one's correct for data type.

File: en/mulesoft/modules/ROOT/pages/ms_composer_ms_release_notes.adoc [hunk 3]
```diff
@@ -1,12 +1,18 @@
...
 
 New versions are automatically available once they are released; you don't need to perform any tasks to move to the new version.
 
 See xref:ms_composer_ms_release_notes_ki.adoc[Known Issues and Limitations for MuleSoft Composer] for a list of known issues and limitations by date.
 
+== September 21, 2023
+
+include::_partials/release-notes/rn-headings.adoc[tag=whatsNew,leveloffset=+3]
+
+* Picklist Mapping is now enabled for all `String` fields. Using Picklist Mapping, you can map data from one application with the corresponding data needed by another application. This allows you to connect two applications that have different values for a similar field and to align the values in those fields across applications. You can determine how the possible values of an output field (data pill) map to the possible values of an input field. You can reuse the mapping across flows.
```
Comment (hannanelson) [inline en/mulesoft/modules/ROOT/pages/ms_composer_ms_release_notes.adoc:14]: See previous.

Comment (hannanelson) [reply en/mulesoft/modules/ROOT/pages/ms_composer_ms_release_notes.adoc:14]: allows > enables?

File: en/salesforce/src/docs/ms_composer_release_notes_2023.adoc [hunk 4]
```diff
@@ -6,6 +6,12 @@ New versions are automatically available once they are released; you don't need
 
 See xref:ms_composer_release_notes_ki.adoc[Known Issues and Limitations for MuleSoft Composer for Salesforce] for a list of known issues and limitations by date.
 
+== September 21, 2023
+
+include::_partials/release-notes/rn-headings.adoc[tag=whatsNew,leveloffset=+3]
+
+* Picklist Mapping is now enabled for all `String` fields. Using Picklist Mapping, you can map data from one application with the corresponding data needed by another application. This allows you to connect two applications that have different values for a similar field and to align the values in those fields across applications. You can determine how the possible values of an output field (data pill) map to the possible values of an input field. You can reuse the mapping across flows.
```
Comment (hannanelson) [inline en/salesforce/src/docs/ms_composer_release_notes_2023.adoc:13]: See prev

**Review by hannanelson — DISMISSED** [review]

File: en/mulesoft/modules/ROOT/pages/ms_composer_flows.adoc [hunk 5]
```diff
@@ -471,7 +471,7 @@ In this example, the *Status* field in a case in Salesforce is mapped to the *St
 When mapping source data fields:
 
 * The source field must be a data pill with a fixed list of values.
-* The target field must be an enumerated xref:ms_composer_custom_expression_editor.adoc#data-types[data type] field.
+* The target field must be an enumerated xref:ms_composer_custom_expression_editor.adoc#data-types[data type] field or any `String` field.
```
Comment (emileeburkeVT) [inline en/mulesoft/modules/ROOT/pages/ms_composer_flows.adoc:474]:
```suggestion
* The target field must be an enumerated xref:ms_composer_custom_expression_editor.adoc#data-types[data type] field or any string field.
```

File: en/mulesoft/modules/ROOT/pages/ms_composer_flows.adoc (hunk 1 above)
Comment (emileeburkeVT) [inline en/mulesoft/modules/ROOT/pages/ms_composer_flows.adoc:483]:
```suggestion
. In an action card, in a field that supports *Pick from List* or any string field, select *Picklist Mapping* from the menu. +
```

File: en/mulesoft/modules/ROOT/pages/ms_composer_flows.adoc (hunk 2 above)
Comment (emileeburkeVT) [reply en/mulesoft/modules/ROOT/pages/ms_composer_flows.adoc:499]: Fixed the other ones and your comment disappeared - it is a field.

File: en/mulesoft/modules/ROOT/pages/ms_composer_ms_release_notes.adoc (hunk 3 above)
Comment (emileeburkeVT) [inline en/mulesoft/modules/ROOT/pages/ms_composer_ms_release_notes.adoc:14]:
```suggestion
* Picklist Mapping is now enabled for all string fields. Using Picklist Mapping, you can map data from one application with the corresponding data needed by another application. This enables you to connect two applications that have different values for a similar field and to align the values in those fields across applications. You can determine how the possible values of an output field (data pill) map to the possible values of an input field. You can reuse the mapping across flows.
```

File: en/salesforce/src/docs/ms_composer_release_notes_2023.adoc (hunk 4 above)
Comment (emileeburkeVT) [inline en/salesforce/src/docs/ms_composer_release_notes_2023.adoc:13]:
```suggestion
* Picklist Mapping is now enabled for all string fields. Using Picklist Mapping, you can map data from one application with the corresponding data needed by another application. This enables you to connect two applications that have different values for a similar field and to align the values in those fields across applications. You can determine how the possible values of an output field (data pill) map to the possible values of an input field. You can reuse the mapping across flows.
```

**Review by hannanelson — APPROVED** [review]

**Pull request merged**